arcgis-ios-sdk-samples/Content Display Logic/Controllers/SourceCodeViewController.swift

```diff
@@ -63,7 +73,12 @@ class SourceCodeViewController: UIViewController, UIAdaptivePresentationControll
     }
     
     func htmlStringForContent(_ content: String) -> String {
-        let cssPath = Bundle.main.path(forResource: "xcode", ofType: "css") ?? ""
+        let cssPath: String
+        if traitCollection.userInterfaceStyle == .dark {
+            cssPath = Bundle.main.path(forResource: "solarized-dark", ofType: "css") ?? ""
+        } else {
+            cssPath = Bundle.main.path(forResource: "xcode", ofType: "css") ?? ""
```

Comment:
Instead of specifying a default value in both places, you could make `cssPath` optional and then specify a default value at the point of use.